### PR TITLE
Add the Caplinger paper.

### PR DESCRIPTION
**Pull request by ltratt**
It seems this was the first conservative scanning GC.

### CODE REVIEW

File: softdev.bib [hunk 1]
```diff
@@ -2871,3 +2871,12 @@ @inproceedings{craciunas08compacting
     month = Jun,
     year = 2008,
 }
+
+@inproceedings{caplinger88memory,
+  author = "Michael A. Caplinger",
```
Comment (jacob-hughes) [inline softdev.bib:2876]: The other entries in the bib do lastname firstname. Should we do this here too?

Comment (ltratt) [reply softdev.bib:2876]: We're all over the shop in that regard. Fortunately, BibTeX treats them as equivalent (except in some unusual cases where you have to use (last, first)).

Comment (jacob-hughes) [reply softdev.bib:2876]: Ok, that works for me then.

**Pull request merged**